instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2BedrockRuntimeTest.java

```diff
@@ -2247,6 +2247,253 @@ void testInvokeModelAnthropicClaude() {
                                 "content", Value.of("Okay, I just said \"This is a test")))));
   }
 
+  @Test
+  void testInvokeModelMetaLlama() {
+    BedrockRuntimeClientBuilder builder = BedrockRuntimeClient.builder();
+    builder.overrideConfiguration(createOverrideConfigurationBuilder().build());
+    configureClient(builder);
+    BedrockRuntimeClient client = builder.build();
+
+    String modelId = "meta.llama3-3-70b-instruct-v1:0";
+
+    Document requestPayload =
+        Document.mapBuilder()
+            .putString("prompt", "Say this is a test")
+            .putNumber("max_gen_len", 10)
+            .putNumber("temperature", 0.8f)
+            .putNumber("top_p", 1)
+            .build();
+
+    SdkJsonGenerator generator = new SdkJsonGenerator(new JsonFactory(), "application/json");
+    DocumentTypeJsonMarshaller marshaller = new DocumentTypeJsonMarshaller(generator);
+    requestPayload.accept(marshaller);
+
+    InvokeModelRequest request =
+        InvokeModelRequest.builder()
+            .modelId(modelId)
+            .body(SdkBytes.fromByteArray(generator.getBytes()))
+            .build();
+
+    InvokeModelResponse response = client.invokeModel(request);
+
+    JsonNode node = JsonNode.parser().parse(response.body().asByteArray());
+    Document responsePayload = node.visit(new DocumentUnmarshaller());
+    assertThat(responsePayload.asMap().get("generation").asString())
+        .isEqualTo("\n\nThis is a test");
+
+    getTesting()
+        .waitAndAssertTraces(
+            trace ->
+                trace.hasSpansSatisfyingExactly(
+                    span ->
+                        span.hasName("chat meta.llama3-3-70b-instruct-v1:0")
+                            .hasKind(SpanKind.CLIENT)
+                            .hasAttributesSatisfying(
```

Comment:
is there a reason not to used `hasAttributesSatisfyingExactly` here and in the other added tests?
```suggestion
                            .hasAttributesSatisfyingExactly(
```

Reply:
The trace span contains several other attributes that aren't Gen AI specific, i.e. `rpc.method`, `http.request.method`, `server.address`, `server.port`, etc. that we don't test.

Reply:
oh right sorry, I knew this 🤦